databend-jdbc/src/test/java/com/databend/jdbc/TestDatabendDriverUri.java

```diff
@@ -8,7 +8,7 @@
 import java.util.Properties;
 
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
-
+@Test(timeOut = 10000)
```

Comment:
Why need timeout in test? The test should pass in specified time.

Reply:
1. in case there is bug in pr which may cause the client to block or loop forever, which is a waste to resources.
2. know which test is blocking from CI， as early as possible